jobwatch/skawatch.py

```diff
@@ -53,6 +59,26 @@ def __init__(self, task, maxage=1, errors=jobwatch.ERRORS, requires=(),
                                           maxage=maxage)
 
 
+class KadiWatch(JobWatch):
+    def __init__(self, task, filename, maxage=1):
+        self.type = 'Application Data'
+        super(KadiWatch, self).__init__(task, filename, maxage=maxage)
```

Comment:
Just use `super().__init__()`. That's the preferred idiom in Python 3.

Reply:
OK.  I assume I still need to pass the args?  (mostly because it didn't work when I didn't)